encrypted-media/drm-mp4-requestmediakeysystemaccess.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
+    <title>Encrypted Media Extensions: Successful Playback, Temporary session with Clear Key, mp4</title>
+    <link rel="help" href="https://w3c.github.io/encrypted-media/">
+
+    <!-- Web Platform Test Harness scripts -->
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+
+    <!-- Helper scripts for Encrypted Media Extensions tests  -->
+    <script src=/encrypted-media/util/utils.js></script>
+    <script src=/encrypted-media/util/fetch.js></script>
+    <script src=/encrypted-media/util/testmediasource.js></script>
+    <script src=/encrypted-media/util/utf8.js></script>
+
+    <!-- Content metadata -->
+    <script src=/encrypted-media/content/content-metadata.js></script>
+
+    <!-- The script for this specific test -->
+    <script src=/encrypted-media/scripts/requestmediakeysystemaccess.js></script>
+
+  </head>
+  <body>
+    <div id='log'></div>
+
+    <script>
+        var contentitem = content['mp4-basic'],
+            config = {  keysystem:          'org.w3.clearkey',
+                        audioType:          contentitem.audio.type,
+                        videoType:          contentitem.video.type,
+                        initDataType:       'keyids'
```

Comment:
Section should look more like:

        var keysystem = getSupportedKeySystem(),
            contentitem = content['mp4-basic'],
            handler = new MessageHandler( keysystem, contentitem ),
            config = {  video:              document.getElementById('videoelement'),
                        keysystem:          keysystem,
                        messagehandler:     handler.messagehandler,
                        audioPath:          contentitem.audio.path,
                        videoPath:          contentitem.video.path,
                        audioType:          contentitem.audio.type,
                        videoType:          contentitem.video.type,
                        initDataType:       contentitem.initDataType,
                        duration:           2 };